loaders/authentication.js

```diff
@@ -0,0 +1,38 @@
+const fetch = require('node-fetch');
+const dotenv = require('dotenv').config();
+
+//base URL of cigarra authentication
+const URL = 'https://sigarra.up.pt/feup/pt/mob_val_geral.autentica?'
+
+//build URL with credentials
+let urlRequest = URL + 'pv_login=' + process.env.UP_USER + '&pv_password=' + process.env.UP_PWD;
+
+let cookie = null;
+
+//Called sigarra api to make authentication
+async function updateCookie(){
```

Comment:
I much prefer arrow function syntax whenever `this` is not necessary.
I understand different preferences, of course.

However, most important of all: we should really define a standard (for this and several other things).
`eslint` and friends would help.

@GambuzX it isn't installed right? We should've had that ready to go beforehand, but maybe can create a task for that and work on it ASAP, otherwise there will be inconsistencies all around. Even in indentation, naming schemes and such there are already some.

I suggest either just using [standard](https://standardjs.com/) to avoid bikeshedding or copying the eslint configs from `nijobs-be`. (If we use standard, let's please use the version with semicolons though :cry:)

Reply:
There is no linter no, I'll create an issue

Reply:
IMO we should prioritize this straight away, and add it to CI, otherwise the PRs will become a bit of a mess.  
I'll try to grab this by tomorrow or something similar if no one picks it up in the interim.